### ISSUE TITLE
Chat Screen

### ISSUE THREAD

**Issue by HimeshNayak** [HimeshNayak/SwingerIOT#6]
Here are some issues generated from Axe-Core DevTools
```
[
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(5) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 3.99 (foreground color: #787878, background color: #eaf7ec, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 15px; font-weight: 500; font-family: Poppins, Arial, sans-serif; color: rgb(120, 120, 120); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 16.8px; width: 315px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 45, 127); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(14) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.59 (foreground color: #ffffff, background color: #64b174, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 12px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 13.6px; width: 31px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 287, 379); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(16) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.59 (foreground color: #ffffff, background color: #64b174, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 15px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 33.6px; width: 253px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 36, 524); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(17) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.59 (foreground color: #ffffff, background color: #64b174, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 12px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 13.6px; width: 31px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 290, 558); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(20) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.59 (foreground color: #ffffff, background color: #64b174, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 15px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 16.8px; width: 76px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 39.5864, 409.8); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(22) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.85 (foreground color: #ffffff, background color: #9c9898, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 15px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 16.8px; width: 286px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 106.54, 456.115); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(25) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.85 (foreground color: #ffffff, background color: #9c9898, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 15px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 16.8px; width: 104px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 263.463, 599.44); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(26) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.85 (foreground color: #ffffff, background color: #9c9898, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 12px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 13.6px; width: 33px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 361.761, 604.16); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:30:02.265Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    }
]
```
